docs/rfc/README.md

```diff
@@ -13,7 +13,7 @@ discussion that might otherwise only be recorded in an ad-hoc way (for example,
 via gists or Google docs) that are difficult to discover for someone after the
 fact. An RFC _may_ give rise to more specific architectural _decisions_ for
 the Cosmos SDK, but those decisions must be recorded separately in
-[Architecture Decision Records (ADR)](../architecture/).
+[Architecture Decision Records (ADR)](../../integrate/architecture).
```

Comment:
This will not work when rendering on GitHub directly. I don't think it needs to change either as RFC and Architecture are in the same category in the new organization.